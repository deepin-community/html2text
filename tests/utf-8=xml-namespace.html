<HTML><HEAD>
<META http-equiv="Content-Type" content="text/html; charset=utf-8">
</HEAD>
<BODY>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black'>+7 
(931) 009-16-22</SPAN></B><B style="mso-bidi-font-weight: normal"><SPAN 
lang=EN-US 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; mso-ansi-language: EN-US'><?xml:namespace 
prefix = "o" ns = "urn:schemas-microsoft-com:office:office" 
/><o:p></o:p></SPAN></B></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>(звонки или 
WhatsApp)<B style="mso-bidi-font-weight: normal"><SPAN 
style="COLOR: red"><o:p></o:p></SPAN></B></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN lang=EN-US 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; mso-ansi-language: EN-US'><A><SPAN 
style="COLOR: blue">Magnum</SPAN><SPAN lang=RU 
style="COLOR: blue; mso-ansi-language: RU">-</SPAN><SPAN 
style="COLOR: blue">magnum</SPAN><SPAN lang=RU 
style="COLOR: blue; mso-ansi-language: RU">2020@mail.ru</SPAN></A></SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue'> 
<o:p></o:p></SPAN></B></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Производитель 
одноразовых медицинских масок, защитных костюмов, халатов и антисептиков 
предлагает <B style="mso-bidi-font-weight: normal"><U>средства индивидуальной 
защиты для предприятий</U></B> по оптовым ценам.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: red'>Наши цены 
вас приятно удивят:<o:p></o:p></SPAN></B></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'>1. 
<SPAN style="COLOR: blue">3-слойные шитые маски</SPAN></SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(спанбонд 40 гр. 3 слоя) - от 13 рублей. <o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>2.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Антисептики собственного производства</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(рецепт ВОЗ, спирт не менее 70%, глицерин, перекись водорода, вода) - от 500 
рублей/литр.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>3.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Антивирусные защитные костюмы "Каспер"</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(спанбонд 25/40/50/60 гр.) - от 155 рублей.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>4.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Защитные костюмы "Тайвек"</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(ламинированный спанбонд 40/60/80 гр.) - от 300 рублей. <o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>5.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Медицинские халаты</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(спанбонд 25/40/50/60 гр.) - от 85 рублей.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: 150%"><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black; LINE-HEIGHT: 150%'>6.</SPAN></B><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; LINE-HEIGHT: 150%'> 
Высокие защитные бахилы <?xml:namespace prefix = "st1" ns = 
"urn:schemas-microsoft-com:office:smarttags" /><st1:metricconverter w:st="on" 
ProductID="40 см">40 см</st1:metricconverter>.</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; LINE-HEIGHT: 150%'> 
(спанбонд 25/40/50/60 гр.) - от 30 рублей. <o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Также у нас есть 
медицинские маски Китайского производства, шапочки, нарукавники, фартуки<B 
style="mso-bidi-font-weight: normal"> <SPAN style="COLOR: red">и любые другие 
средства защиты</SPAN>.<o:p></o:p></B></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue'>Мы 
являемся производителем</SPAN></B><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>, поэтому наши цены 
ниже конкурентов.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Вся продукция 
сертифицирована.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Возможна доставка по 
договоренности в любой регион.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Для постоянных 
партнеров и на крупные объемы цены еще ниже.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Работаем за 
наличный/безналичный расчет с НДС.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: red'>Товар от 
производителя.<o:p></o:p></SPAN></B></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>Производство и склад 
в г. Новосибирск. Образцы товаров в наличии в Москве.<o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 15pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal style="MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal"><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p>&nbsp;</o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><B style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 16pt; FONT-FAMILY: "Arial","sans-serif"'>Позвоните нам или 
напишите в WhatsApp:</SPAN></B><SPAN 
style='FONT-SIZE: 16pt; FONT-FAMILY: "Arial","sans-serif"'> <BR></SPAN><B 
style="mso-bidi-font-weight: normal"><SPAN 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: black'>+7 
(931) 009-16-22</SPAN></B><SPAN 
style='FONT-SIZE: 18pt; FONT-FAMILY: "Arial","sans-serif"'><o:p></o:p></SPAN></DIV>
<DIV class=MsoNormal 
style="TEXT-ALIGN: center; MARGIN: 0cm 0cm 0pt; LINE-HEIGHT: normal" 
align=center><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'>или напишите на 
почту: </SPAN><SPAN lang=EN-US 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"; COLOR: blue; mso-ansi-language: EN-US'><A><SPAN 
style="COLOR: blue">Magnum</SPAN><SPAN lang=RU 
style="COLOR: blue; mso-ansi-language: RU">-</SPAN><SPAN 
style="COLOR: blue">magnum</SPAN><SPAN lang=RU 
style="COLOR: blue; mso-ansi-language: RU">2020@mail.ru</SPAN></A></SPAN><SPAN 
lang=EN-US style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'> 
</SPAN><SPAN 
style='FONT-SIZE: 14pt; FONT-FAMILY: "Arial","sans-serif"'><o:p></o:p></SPAN></DIV></BODY></HTML>
