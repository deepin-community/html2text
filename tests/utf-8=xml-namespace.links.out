utf-8=xml-namespace.default.out